src/dataworker/Dataworker.ts

```diff
@@ -1246,17 +1180,7 @@ export class Dataworker {
 
           const outputAmount = sdkUtils.getRelayDataOutputAmount(slowFill.relayData);
```

Comment:
It probably pays to revert to direct dereferencing where possible, since that eliminates an additional call to `isV3...()`

```suggestion
          const { outputAmount } = slowFill.relayData;
```